fedimint-server/src/config.rs

```diff
@@ -168,14 +168,26 @@ impl ServerConfigConsensus {
         })
     }
 
-    pub fn to_client_config(&self, module_config_gens: &ModuleConfigGens) -> ClientConfig {
+    pub fn to_client_config(&self, module_config_gens: &ModuleInitRegistry) -> ClientConfig {
         self.to_client_config_try(module_config_gens)
             .expect("configuration mismatch")
     }
 }
+use impl_tools::autoimpl;
 
-pub type ModuleConfigGens =
-    BTreeMap<&'static str, Arc<dyn FederationModuleConfigGen + Send + Sync>>;
+// TODO: turn into newtype
+pub type DynModuleInit = Arc<dyn ModuleInit + Send + Sync>;
```

Comment:
According to our convention we might want to call it `IModuleInit`?

Reply:
`IModuleInit` would be for the `trait IModuleInit` which is called `ModuleInit` Rn ... :D